{{ partial "header.html" . }}

<div id="headerwrap">
  <div class="container">
    <div class="row centered">
      <div class="col-lg-12">
        <h1>Jason M Knight</h1>
        <h3>Computational Biologist &nbsp|&nbsp <a href="mailto:jason@jasonknight.us">jason@jasonknight.us</a></h3>
      </div>
      <!--/.col-lg-12 -->
    </div>
    <!--/.row -->
  </div>
  <!--/.container -->

</div>
<!--/.#headerwrap -->

<div class="container desc">
  <div class="row">

    <div class="col-lg-2 col-lg-offset-1">
      <h5>ABOUT</h5>
    </div>
    <div class="col-lg-7">
      <p>
        I'm a computational biologist/engineer/bioinformatician with several years experience
        with high-dimensional data analysis. I have a penchant for programming and
        programming language design with some hardware and entrepreneurial experience
        mixed in. I also have a wide range of additional interests and hobbies (cryptocurriencies,
        biohacking, etc.).
      </p>

      <p>
        I am currently finishing my Ph.D. at Texas A&amp;M University in the <a href="http://gsp.tamu.edu/">Genomic Signal Processing laboratory</a>        where I apply tools from modern engineering, mathematics, and statistics
        in order to understand and treat cancer. </p>

      <p>
        Specifically, I have worked closely with biologists to develop Bayesian statistical
        models to uncover multivariate interactions among genes from RNA-Seq data.
        This allows us to classify types of cancer better than nonlinear SVM methods
        (which themselves are typically considered state of the art). I have also
        modeled biological regulatory networks using probabilistic Markov models
        that incorporate pathway knowledge from the biology literature as well as
        high-throughput data.
      </p>

      <p>
        In the future, I want to continue using my knowledge and skills to enable principled
        approaches towards the detection, classification, and treatment of human
        diseases. </p>


    </div>
    <div class="col-lg-2">
      <p><a class="btn btn-info" href="/static/jknight_resume_2014.pdf"><i class="fa fa-file"></i><sm>&nbsp; One Page Resume </sm></a>
      </p>
      <p><a class="btn btn-info" href="#"><i class="fa fa-file"></i><sm>&nbsp; Full CV (coming soon)</sm></a>
      </p>
    </div>

  </div>
  <!--/.row -->
</div>
<!--/.container -->

<hr>

<!--PORTFOLIO DESCRIPTION -->

<div class="container desc">
  <div class="row">
    <div class="col-lg-2 col-lg-offset-1">
      <h5>RESEARCH/<br>PROJECTS</h5>
    </div>

    <div class="col-lg-7">
      <img src="/img/nfkb.png" alt="nfkb project image" class="img-responsive">
    </div>

    <div class="col-lg-9 col-lg-offset-3">
      <t>Genetic Regulatory Network Modeling Using Biological Pathway Knowledge</t>
      <p>
      Every single one of the <a href="http://www.ncbi.nlm.nih.gov/pubmed/23829164">40 trillion cells</a> in your body has tens
      of thousands of genes which are coordinated in a chaotic symphony of
      regulation for your continued living (homeostasis in biological
      parlance). For decades now, biologists have been studying this regulatory
      network piece by piece through a variety of ingenious techniques and a
      mind-boggling amount of effort.
      </p>
      
      <p>
      The trouble is, this biological pathway knowledge is incomplete and
      sometimes contradictory. Due to this, building predictive models from
      this knowledge is difficult. In this work, we used a modeling approach
      embracing the uncertainty through a family of Markov chains (which
      themselves are a random dynamical system model).  </p>

      <p>
      Specifically, we obtained pathways from the biological literature
      regarding the NF-kB network which is central to cellular inflammatory
      processes. Then constructing a family of Markov chains from these
      pathways, we could then make predictions of how the network might evolve
      under the deletion of several genes under the effect of various stimuli.
      These predictions were then qualitatively validated against additional
      biological literature where these experiments were carried out on mouse
      gene knockout models.
      </p>

      <p>
      For more information, please consult <a href="/static/knight2012generating.pdf">our paper</a>, and consult my <a href="https://github.com/binarybana/grn-pathways">open source Haskell code</a> used to generate the family of Markov models.
      </p>
      <p class="text-center">§</p>
    </div>

    <div class="col-lg-7 col-lg-offset-3">
      <img src="/img/robert.png" alt="nonlinear fit project image" class="img-responsive">
    </div>

    <div class="col-lg-9 col-lg-offset-3">
      <t>Nonlinear fit of flourescence data</t>
      <p>...</p>
    </div>

    <div class="col-lg-9 col-lg-offset-3">
      <i>More projects coming soon...</i>
    </div>
  </div>
  <!--/.row -->
</div>
<!--/.container -->

<hr>
<div class="container desc">
  <div class="row">
    <div class="col-lg-2 col-lg-offset-1">
      <h5>BLOG</h5>
    </div>
    <div class="col-lg-9">
      My ten most recent posts:
      <ul class="list-unstyled">
        {{ range first 10 .Data.Pages }} {{ .Render "li"}} {{ end }}
        <!--<li><a href="/post">See a listing of all posts here</a>.</li>-->
        <!--<li>...</li>-->
      </ul>
        <a href="/post" class="btn btn-info">See a listing of all posts here</a>
    </div>
  </div>
  <!--/.row -->
</div>
<!--/.container -->

<hr>

<div class="container desc">
  <div class="row">

    <div class="col-lg-2 col-lg-offset-1">
      <h5>EDUCATION</h5>
    </div>
    <div class="col-lg-6">
      <p>
        <t>Ph.D. Electrical Engineering</t>
        <br/>
        Texas A&M University
        <br/>
        <i>Advisor: Edward Dougherty</i>
        <br/>
        May 2015 (Expected)
      </p>
    </div>

    <div class="col-lg-6 col-lg-offset-3">
      <p>
        <t>B.S. Biomedical Engineering</t>
        <br/>
        Texas A&M University
        <br/>
        <i>Minor: Mathematics (summa cum laude)</i>
        <br/>
        May 2009
      </p>
    </div>

  </div>
  <!--/.row -->
</div>
<!--/.container -->

<hr>

<div class="container desc">
  <div class="row">
    <div class="col-lg-2 col-lg-offset-1">
      <h5>SKILLS</h5>
    </div>
    <div class="col-lg-9">
      <t>Programming</t>
      <ul>
        <li>Proficient: <a href="http://julialang.org/">Julia</a>, Python, Haskell </li>
        <li>Working-level: C, Javascript </li>
        <li>In the wings: R, C++, Bash </li>
      </ul>
    </div>

    <div class="col-lg-9 col-lg-offset-3">
      <t>Bioinformatics</t>
      <ul>
        <li><a href="https://www.youtube.com/watch?v=76fzkSUX1NU">RNA-Seq</a> 
          <ul>
            <li>Alignment</li>
            <li>Quantification</li>
            <li>Single Cell</li>
            <li>Differential Expression</li>
            <li>Multivariate Analysis</li>
            <li>Pattern Detection</li>
          </ul>
        <li>ChIP-Seq</li>
        <li>RT-qPCR</li>
      </ul>
    </div>
    <div class="col-lg-9 col-lg-offset-3">
      <t>Statistics</t>
      <ul>
        <li>Bayesian inference</li>
          <ul>
            <li>Classification</li>
            <li>Bayesian networks</li>
            <li>Model-based quantization</li>
          </ul>
        <li>Markov Chain Monte Carlo techniques (including <a href="http://www.stat.tamu.edu/~fliang/"><abbr title="Stochastic Approximation Markov Chain Monte Carlo">SAMCMC</abbr></a>) </li>
        <li>Monte Carlo Integration </li>
      </ul>
    </div>

  </div>
  <!--/.row -->
</div>
<!--/.container -->

<hr>

<div class="container desc">
  <div class="row">
    <div class="col-lg-2 col-lg-offset-1">
      <h5>AWARDS/<br>PUBLICATIONS</h5>
    </div>
    <div class="col-lg-9">
      <t>Awards</t>
      <ul>
        <li> 2010 - Innovative Signal Analysis Fellowship </li>
        <li> 2010 - National Science Foundation Graduate Fellowship - Two Time Honorable Mention </li>
        <li> 2009 - Texas A&M Electrical and Computer Engineering Departmental Scholarship </li>
        <li> 2008 - Barry M. Goldwater Scholar - Two Time Honorable Mention </li>
        <li> 2008 - <a href="http://maysbusiness.tamu.edu/index.php/aggie-student-consultants-solve-real-world-problems/">1<sup>st</sup> place in Mays Business School Consulting Competition </a></li>
        <li> 2007 - Texas A&M University Scholar </li>
        <li> 2005 - Texas A&M Presidential Endowed Scholarship </li>
      </ul>
    </div>
    <!--/.col-lg-12-->

    <div class="col-lg-9 col-lg-offset-3">
      <t>Publications</t>
      <ul>
        <li>Jason Knight, Aniruddha Datta, and Edward Dougherty. 
        <a href="/static/knight2012generating.pdf">“Generating Stochastic Gene Regulatory Networks Consistent with Pathway
          Information and Steady-State Behavior.”</a> <em>Biomedical Engineering,
        IEEE Transactions on</em> 59 (6). IEEE: 1701–1710, 2012.</li>

        <li>Jason Knight, Laurie Davidson, Damir Herman, Camilia Martin, Jennifer Goldsby,
        Ivan Ivanov, Sharon Donovan, and Robert Chapkin. 
        <a href="http://www.nature.com/srep/2014/140626/srep05453/full/srep05453.html">“Non-Invasive
        Analysis of Intestinal Development in Preterm and Term Infants Using
        RNA-Sequencing.”</a><em>Scientific Reports</em>, 4. Nature Publishing
        Group. 2014.</li>

        <li>Jason Knight, Ivan Ivanov, and Edward Dougherty. <a href="#not-available-yet">“The
        Effectiveness of Prior Distributions with Network Models in Systems
        Biology.”</a> <em>Under Preparation</em>, 2014.</li>
        
      <li>Jason Knight, Ivan Ivanov, and Edward Dougherty. <a href="#not-available-yet">“MCMC
        Implementation of the Optimal Bayesian Classifier for Non-Gaussian
        Models: Model-Based RNA-Seq Classification.” </a><em>Under
        Review</em>, 2014.</li>

        <li>Jason Knight, Ivan Ivanov, Karen Triff, Robert Chapkin, and Edward
        Dougherty. <a href="#not-available-yet">“Detecting Multivariate Gene Interactions in RNA-Seq
          Data Using Optimal Bayesian Classification.”</a> <em>Under
        Review</em>, 2014.</li>
    </ul>
  </div>
  <!--/.col-lg-12-->

  </div>
  <!--/.row -->
</div>
<!--/.container -->

{{ partial "footer.html" . }}
